api/client/connection.go

```diff
@@ -368,11 +370,18 @@ func (c *Connection) LogPostPath(ctx context.Context, space api.SpaceID, opts *L
 	return payloads.Error()
 }
 
-func (c *Connection) LogPostPathStream(ctx context.Context, space api.SpaceID, opts *LogPostOpts, paths ...string) (*Stream, error) {
-	body := api.LogPostRequest{Paths: paths}
-	if opts != nil {
-		body.JSONTypeConfig = opts.JSON
-		body.StopErr = opts.StopError
+func (c *Connection) LogPostPathStream(ctx context.Context, space api.SpaceID, opts LogPostOpts, paths ...string) (*Stream, error) {
```

Comment:
I would just assume keep this option a pointer so you don't have to do `api.LogPostOpts` in the vanilla post case.

Reply:
Done in 9cfce89f151ce9b9497e4ffa92f5215b6c1c2f4e.

(I think passing a zero-valued struct is preferable to passing a nil, but that's best kept to a separate change)